compiler/executable/match_/planner/plan.rs

```diff
@@ -72,6 +72,12 @@ pub const AVERAGE_QUERY_OUTPUT_SIZE: f64 = 1.0; // replace with actual statistic
 pub const AVERAGE_STEP_COST: f64 = 1.0; // replace with actual heuristic
 pub const VARIABLE_PRODUCTION_ADVANTAGE: f64 = 0.05; // this is a percentage 0.00 <= x < 1.00
 
+typedb_error! {
+    pub QueryPlanningError(component = "Query Planner", prefix = "QPL") {
+        ExpectedPlannableConjunction(1, "Planning failed as no valid pattern ordering as been found"),
```

Comment:
```suggestion
        ExpectedPlannableConjunction(1, "Planning failed as no valid pattern ordering has been found"),
```

Reply:
I think messages like these should have something to the tune of "this is a bug!"